client/state/sites/test/selectors.js

```diff
@@ -1184,4 +1194,675 @@ describe( 'selectors', () => {
 			expect( frontPageType ).to.eql( 'page' );
 		} );
 	} );
+
+	describe( '#canJetpackSiteManage()', () => {
+		it( 'it should return `null` for a non-existing site', () => {
+			const state = {
+				sites: {
+					items: {}
+				}
+			};
+			let siteId;
+
+			const canManage = canJetpackSiteManage( state, siteId );
+			expect( canManage ).to.equal( null );
+		} );
+
+		it( 'it should return `null` for a non jetpack site', () => {
+			const siteId = 77203074;
+
+			const state = {
+				sites: {
+					items: {
+						77203074: {
```

Comment:
This code is using const as a property value, so it doesn't change much here if you would use it also as a key to represent the same value. I agree that static data is very useful in tests, but mostly in assertions where we want to make sure that it always behaves the same.